meetings/agenda.md

```diff
@@ -7,3 +7,48 @@ Please file pull requests to add, or discuss items to add, to the agenda.
 
 ## Items to discuss
 
+### Components to be potentially moved to security-only status
+
+Proposal: these components should (probably) be clearly labeled so that maintainers know that only security
+issues are to be fixed, if they occur.
+
+ * laminas-db
+ * laminas-developer-tools
+ * laminas/laminas-oauth
+ * laminas/laminas-xml
+ * laminas/laminas-composer-autoloading
+ * laminas/laminas-auradi-config
+ * laminas/laminas-development-mode
+ * laminas/documentation
+ * laminas/laminas-barcode
+ * laminas/laminas-config-aggregator-parameters
+ * laminas/laminas-config
+ * laminas/laminas-config-aggregator-modulemanager
+ * laminas/laminas-console
+ * laminas/laminas-dom
+ * laminas/laminas-crypt
+ * laminas/laminas-mvc-console
+ * laminas/laminas-json
+ * laminas/laminas-http
+ * laminas/laminas-memory
+ * laminas/laminas-loader
+ * laminas/laminas-servicemanager-di
+ * laminas/laminas-json-server
+ * laminas/laminas-mime
```

Comment:
laminas-mime and laminas-mail go together. There have been quite a few patches submitted, many of which providing support for more header and content types. The two are used fairly widely (which surprises me!), particularly for _parsing mailboxes and mail_, and I'm not sure if it makes sense to put them in security-only mode.

Reply:
yes. please not, as there is no other complete alternative for parsing input emails.

for composing emails, there are alternatives, but not for parsing (loading)

Reply:
They will not be abandoned or forgotten, please read this reply https://github.com/laminas/technical-steering-committee/pull/31#issuecomment-646657522